handbook/superset.md

```diff
@@ -0,0 +1,88 @@
+# Superset
+
+## What is Superset
+Superset is a way to query, visualize, and explore the data. Currently, the data hosted on Superset uses Postgres as the meta database engine.
+
+## Accessing Superset in Web Browser
+In order to access Superset, you would first need access to FASSE and the NSAPH host (instructions [here](fasse.md))
+
+Open Terminal, and enter the following command: 
+```
+ssh -L8088:localhost:8088 username@nsaph.rc.fas.harvard.edu
+```
+After logging in through this specific port, you can access the Superset interface locally. In your browser, enter the following url: `http://localhost:8088/login/`
+
+Enter your Superset credentials.
+```{warning}
+If you don't know your Superset credentials, stop immediately and contact us.
+```
+
+Once the Superset authentication is completed, you can now explore the datasets you are granted access to. Start by going to SQL Editor under SQL Lab.
+
+```{figure} imgs/superset_welcome.png
+---
+scale: 30%
+align: center 
+---
+Accessing SQL Lab.
+```
+
```

Comment:
Add a `{warning}` that the 'download data' button retrieves the data locally and that one should not click it if they are querying medical data.